CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ option(ADDRESS_SANITIZER "Build with AddressSanitizer." OFF)
 option(MEMORY_SANITIZER "Build with MemorySanitizer." OFF)
 option(THREAD_SANITIZER "Build with ThreadSanitizer." OFF)
 option(UNDEFINED_BEHAVIOR_SANITIZER "Build with UndefinedBehaviorSanitizer." OFF)
+option(HEAP_PERF_TESTS "Build with heap perf tests." OFF)
+option(BUILD_DEBUG_HEAP "Build debug heap with guard bands and validation." OFF)
+option(ALIGNED_MEMORY_MODEL "Aligned memory model ONLY." OFF)
```

Comment:
Don't think so. 

HEAP_PERF_TESTS run for a while and might produce weird results in Travis environment.
BUILD_DEBUG_HEAP will run two orders of magnitude slower and will fail tests due to bloat size of the allocations - this is expected
ALIGNED_MEMORY_MODEL - most devices will not need aligned access on the heap and will save on memory and performance. The tests, on the other hand, are compiled with this flag on.

Reply:
This comment is for ALIGNED_MEMORY_MODEL. Just wonder if we should just provide customer with default aligned access. With this being OFF, they will still need to contact us for help when they hit bus error on the device when they put the code on different platforms, but if it is ON, most of these failures can be avoid. It might be OK to note down the solution in trouble shooting guide, but I wonder if default to be ON would make life easier for both us and them.

Reply:
True but vast majority of the customers will not have degraded perf and more fragmentation